examples/user_guide/Colormaps.ipynb

```diff
@@ -72,9 +72,9 @@
     "    n = len(cms)*1.0\n",
     "    c=ceil(n/cols) if n>cols else cols\n",
     "    bars = [hv.Image(spacing, ydensity=1, label=\"{0} ({1})\".format(r.name,r.provider))\\\n",
-    "            .options(cmap=process_cmap(r.name,provider=r.provider), **opts)\n",
+    "            .opts(cmap=process_cmap(r.name,provider=r.provider), **opt_kwargs)\n",
```

Comment:
I think I would prefer to inline those options instead of using `**`.

Reply:
Seems way too long in this case.